components/CodeSharePost.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Link from 'next/link';
+
+// helpers
+import { getDate } from '@helpers/data-formatter';
+
+// Models
+import { CodeSharePostInterface } from '@models/CodeShare.model';
+
+export const CodeSharePost = ({ data }: { data: CodeSharePostInterface }): JSX.Element => {
+    return (
+        <div className="mb-14 pr-10">
+            <Link href={`/codeshare/${data.urlTitle}`}>
+                <a className="font-bold inline-block mb-1 no-underline text-blue-500 text-lg hover:underline">
+                    <h3 className="m-0">{data.title}</h3>
+                </a>
+            </Link>
```

Comment:
```suggestion
        <h3 className="m-0">
            <Link href={`/codeshare/${data.urlTitle}`}>
                <a className="font-bold inline-block mb-1 no-underline text-blue-500 text-lg hover:underline">
                    {data.title}
                </a>
            </Link>
        </h3>
```

You can't have an h3 inside an a is not valid HTML 5.